dev-python/terminado/terminado-0.8.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Terminals served to term.js using Tornado websockets"
+HOMEPAGE="https://pypi.python.org/pypi/terminado https://github.com/jupyter/terminado"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND="
+	dev-python/ptyprocess[${PYTHON_USEDEP}]
+	>=www-servers/tornado-0.4[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
```

Comment:
Didn't I just tell you py.test is not necessary either and you should be able to just `python tests/sumthingsumthing.py`?

Reply:
Actually you told that nose is not necessary and that upstream calls pytest, but than this was maybe a misunderstanding. I will take a look.

Reply:
Just running `"${PYTHON}" terminado/tests/basic_test.py || die` causes some trouble:

```
 * python3_4: running distutils-r1_run_phase python_test
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
ERROR:tornado.application:Uncaught exception in /named/3
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/tornado/websocket.py", line 412, in _run_callback
    callback(*args, **kwargs)
  File "/var/tmp/portage/dev-python/terminado-0.8.1/work/terminado-0.8.1-python3_4/lib/terminado/websocket.py", line 54, in open
    self.terminal = self.term_manager.get_terminal(url_component)
  File "/var/tmp/portage/dev-python/terminado-0.8.1/work/terminado-0.8.1-python3_4/lib/terminado/management.py", line 299, in get_terminal
    raise MaxTerminalsReached(self.max_terminals)
terminado.management.MaxTerminalsReached: Cannot create more than 3 terminals
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
ERROR:tornado.application:Uncaught exception in /unique
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/tornado/websocket.py", line 412, in _run_callback
    callback(*args, **kwargs)
  File "/var/tmp/portage/dev-python/terminado-0.8.1/work/terminado-0.8.1-python3_4/lib/terminado/websocket.py", line 54, in open
    self.terminal = self.term_manager.get_terminal(url_component)
  File "/var/tmp/portage/dev-python/terminado-0.8.1/work/terminado-0.8.1-python3_4/lib/terminado/management.py", line 265, in get_terminal
    raise MaxTerminalsReached(self.max_terminals)
terminado.management.MaxTerminalsReached: Cannot create more than 3 terminals
terminado/tests/basic_test.py:239: DeprecationWarning: Please use assertEqual instead.
  self.assertEquals(msg, None)
.WARNING:tornado.general:tornado.autoreload started more than once in the same process
.
----------------------------------------------------------------------
Ran 8 tests in 34.709s
```

Executing the tests with py.test just works fine and actually I don't think it is a good idea to patch the tests to not open more than three terminals.

Reply:
Ok, thanks for testing. Just please add `-v` then.

Reply:
Added -v and further bumped to EAPI=6